docs/admin/runai-setup/self-hosted/k8s/backend.md

```diff
@@ -38,70 +36,79 @@ Run the helm command below:
 !!! Tip
     Use the  `--dry-run` flag to gain an understanding of what is being installed before the actual installation.
 
-### Additional configurations (optional)
 
-There may be cases where you need to set additional properties as follows:
+## Additional Run:ai configurations (optional)
+There may be cases where you need to set additional properties,
+To apply the changes run `helm upgrade` and use `--set` to set specific confiogurations, and restart the relevant Run:ai pods so they can fetch the new configurations.
 
 |  Key     | Change   | Description |
 |----------|----------|-------------| 
-| `redis.auth.password` | Redis (Runai internal cache mechanism) applicative password | Override the default password |
-| `keycloakx.adminUser` | User name of the internal identity provider administrator | This user is the administrator of Keycloak | 
-| `keycloakx.adminPassword` | Password of the internal identity provider administrator | This password is for the administrator of Keycloak | 
-| `global.keycloakx.host` |  KeyCloak (Run:ai internal identity provider) host path | Override the DNS for Keycloak. This can be used to access Keycloak from outside the Run:ai Control Plane cluster via ingress | 
 | `global.ingress.ingressClass` |  Ingress class  |  Run:ai default is using NGINX. If your cluster has a different ingress controller, you can configure the ingress class to be created by Run:ai |
 | `global.ingress.tlsSecretName`  | TLS secret name  | Run:ai requires the creation of a secret with domain certificate. See [above](#domain-certificate). If the `runai-backend` namespace already had such a secret, you can set the secret name here  |
```

Comment:
@ozRunAI not sure where this is supposed to link to. Please try not to use "here" and "above" when linking, use the terms the reader needs.